config/config.yaml

```diff
@@ -83,7 +83,7 @@ enrichment:
     # if activated, you need to provide a GMT file with gene sets of interest
     gene_sets_file: "resources/gene_sets/dummy.gmt"
     fdr_gene_set: 0.05
-    nperm: 100000
+    eps: 0.0
```

Comment:
Since this changes here, the config schema has to be changed as well. It appears there's not test that covers this case of an incorrect schema?

Reply:
I just forgot to update the two relevant places for this in sync: `workflow/schemas/config.schema.yaml` AND `.test/config/config.yaml`.
If you forget both, the test won't complain... Oups.

Reply:
But this should be resolved with the following 3 commits. Which shows that I also forgot to update a third place, that the linting now found...

Reply:
Make it 4.